Projects/Feature/BottleStorage/Interface/Sources/PingPongDetail/View/PhotoSharePingPongView.swift

```diff
@@ -7,27 +7,35 @@
 
 import SwiftUI
 
+import DomainBottleInterface
+
 import SharedDesignSystem
 
 public struct PhotoSharePingPongView: View {
   private let isActive: Bool
   private let pingPongTitle: String
-  private let photoShareState: PhotoShareStateType
+  private let photoShareState: PingPongPhotoStatus
+  private let myImageURL: String?
+  private let otherImageURL: String?
```

Comment:
```suggestion
  private let myProfileImageURL: String?
  private let peerProfileImageURL: String?
```

View랑 네이밍 통일시키면 더 좋을 거 같아!

Reply:
이거 profile 추가해주는게 명명 더 명확해 보여서 entity도 명명 반영해서 수정했어!